### ISSUE TITLE
NullReferenceException on start up

### ISSUE THREAD

**Issue by FlameSoulis** [resonite-modding-group/ResoniteModLoader#5]
Tested with 2.5.1 and 2.5.0: attempting to launch with Mod Loader will result in the following:

```
8:20:30 PM.172 ( -1 FPS)    [DEBUG][ResoniteModLoader] Changing compatibility hash from g6O6jx7g5Oa8/J00KXMzFw== to O3418o7DkChI1EzcPksTCg==
8:20:30 PM.172 ( -1 FPS)    [INFO] [ResoniteModLoader] Compatibility hash spoofing succeeded
8:20:30 PM.174 ( -1 FPS)    [INFO] [ResoniteModLoader] Loading assemblies from rml_mods
8:20:30 PM.343 ( -1 FPS)    [DEBUG][ResoniteModLoader] Method "System.Void FrooxEngine.Engine::RequestShutdown()" has been patched by "com.resonitemodloader.ResoniteModLoader"
8:20:30 PM.371 ( -1 FPS)    [ERROR][ResoniteModLoader] Exception in execution hook!
System.NullReferenceException: Null method for com.resonitemodloader.ResoniteModLoader
  at HarmonyLib.PatchProcessor.Patch () [0x0001d] in <255414689d344b1385ae719f6cce8c55>:0 
  at HarmonyLib.Harmony.Patch (System.Reflection.MethodBase original, HarmonyLib.HarmonyMethod prefix, HarmonyLib.HarmonyMethod postfix, HarmonyLib.HarmonyMethod transpiler, HarmonyLib.HarmonyMethod finalizer) [0x00028] in <255414689d344b1385ae719f6cce8c55>:0 
  at ResoniteModLoader.AssemblyHider.PatchResonite (HarmonyLib.Harmony harmony, System.Collections.Generic.HashSet`1[T] initialAssemblies) [0x000f1] in <0b5d7d2085bb4c51ae447d5c47bc968f>:0 
  at ResoniteModLoader.HarmonyWorker.LoadModsAndHideModAssemblies (System.Collections.Generic.HashSet`1[T] initialAssemblies) [0x00010] in <0b5d7d2085bb4c51ae447d5c47bc968f>:0 
  at ResoniteModLoader.ExecutionHook..cctor () [0x00070] in <0b5d7d2085bb4c51ae447d5c47bc968f>:0 
```

Occurs with or without mods in the mods folder.
[DEMONDOGGO-PC - 2023.11.28.1365 - 2023-12-01 20_33_29.log](https://github.com/resonite-modding-group/ResoniteModLoader/files/13533112/DEMONDOGGO-PC.-.2023.11.28.1365.-.2023-12-01.20_33_29.log)

**Comment by AlexW-578** [comment]
I had the same issue, even when redownloading the dll files. 
However, i managed fixed it by recompiling the loader again locally.

**Comment by XDelta** [comment]
A fix has already been made, just isn't in a build yet. It doesn't stop the mods from being loaded.

**Comment by XDelta** [comment]
Fix included in 2.6.0
